src/gui/widgets/settings/ankisettingshelp.ui

```diff
@@ -724,7 +729,28 @@ These will expand to larger expressions in the final card.</string>
            </property>
           </widget>
          </item>
-         <item row="26" column="1">
+         <item row="21" column="1">
+          <widget class="QLabel" name="labelTermPitchCategoriesEx">
+           <property name="text">
+            <string>Comma-separated list of pitch accent categories for the term (e.g. heiban,kifuku,atamadaka,nakadaka,odaka)</string>
```

Comment:
Comma-separated list of pitch accent categories for the term: heiban, kifuku, atamadaka, nakadaka, odaka

I don't think there is a need to show what the marker could actually look like in text.